server/coin-scraper.js

```diff
@@ -86,15 +86,16 @@ module.exports = class CoinScraper {
     this.flagNewArrivals();
   }
 
-  async startLookingForNewArrivals () {
+  async startLookingForNewArrivals() {
     try {
       const response = await this.fetchMostRecentListedItems();
       this.mostRecentItems = response.data;
       this.mostRecentItems = Array.isArray(this.mostRecentItems) ? this.mostRecentItems : [];
       this.updateDictionary();
       this.flagNewArrivals();
-    } catch(err) {
-      console.log(err);
+    } catch (err) {
+      const errorTiming = moment().format('hh:mm:ss');
```

Comment:
@Shapokhan `hh` in moment format is meant for 12 hour clock format, so if time is 15 minutes past midnight, `hh:mm` would show 12:30. `HH` on the other hand represents a 24 hour clock so `HH:mm` for the same time mentioned before would be 00:30. I think we should go for 24 hour clock format, what do think ? Also, let's add the date to this time as well to make error logging even more helpful.

Reply:
@Shapokhan i set it to moment().toString() which seems to have a nice format, check it here: https://momentjs.com/docs/#/displaying/as-string/.